learn/getting_started/installation.mdx

```diff
@@ -152,7 +152,7 @@ chmod +x meilisearch
 
 ### Meilisearch Cloud
 
-[Meilisearch Cloud](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=docs&utm_medium=installation-guide) is one of the easiest way to get started with Meilisearch. The Build plan allows you to index up to 100k documents and perform 10k search requests per month for free!
+[Meilisearch Cloud](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=docs&utm_medium=installation-guide) greatly simplifies installing, using, and maintaining Meilisearch. [Get started with a 14-day free trial](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=docs&utm_medium=installation-guide).
```

Comment:
Trying to address @gmourier's request to explicit mention updates. Removed "using" because it felt too vague and less relevant. Teplaced it with "updating".

```suggestion
[Meilisearch Cloud](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=docs&utm_medium=installation-guide) greatly simplifies installing, maintaining, and updating Meilisearch. [Get started with a 14-day free trial](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=docs&utm_medium=installation-guide).
```

Reply:
Perfect @guimachiavelli, thanks!